Documentation/contributor-guide/community-membership.md

```diff
@@ -28,7 +28,7 @@ Members are continuously active contributors in the community.  They can have
 issues and PRs assigned to them. Members are expected to remain active 
 contributors to the community.
 
-**Defined by:** Member of the etcd GitHub organization
+**Defined by:** Member of the etcd GitHub organization, added via [membership nomination] issue template.
```

Comment:
It's not defined by it, please add it to requirements like in https://github.com/kubernetes/community/blob/master/community-membership.md

Reply:
Sure thing - have adopted k/org style approach in requirements 👍🏻